app/src/main/AndroidManifest.xml

```diff
@@ -93,6 +93,13 @@
                 <action android:name="org.apache.fineract.JobsReceiver" />
             </intent-filter>
         </receiver>
+
+        <receiver android:name=".utils.NetworkChangeReceiver">
+            <intent-filter>
+                <action android:name="android.net.conn.CONNECTIVITY_CHANGE" />
+            </intent-filter>
+        </receiver>
+
```

Comment:
These lines of code are redundant. There is no need to register broadcaster receive in manifest if we are doing it dynamically within the activity. Please take a look at this [PR](https://github.com/openMF/mifos-mobile/pull/1494/commits/5c6430883c28a9074a78b42c08a5b2f01fea59c4) of Shiwangi Singg (GSoC student), she has done similar work.